sbpy/activity/gas/productionrate.py

```diff
@@ -1,24 +1,30 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
+===========================
 sbpy Production Rate Module
+===========================
+
+:author: Giannina Guzman (gguzman2@villanova.edu)
 
 created on June 26, 2019
 """
 
+import tempfile
 import numpy as np
 import astropy.constants as con
 import astropy.units as u
 from astropy.time import Time
 from astroquery.jplhorizons import Horizons, conf
 from astroquery.jplspec import JPLSpec
+from astroquery.lamda import Lamda
 from ...bib import register
 from ...data import Phys
 
 conf.horizons_server = 'https://ssd.jpl.nasa.gov/horizons_batch.cgi'
```

Comment:
This conf line seems to be unnecessary. horizons_server should be fixed in recent releases of astroquery and we are not using `astroquery.jplhorizons` directly here.